setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='txrudp',
-    version='0.4.2',
+    version='0.5.0',
```

Comment:
Please change the version [here](https://github.com/OpenBazaar/txrudp/blob/master/txrudp/__init__.py#L1), as well.